### ISSUE TITLE
Add all contributor bot to brainhack jupyter book

### ISSUE THREAD

**Issue by htwangtw** [brainhackorg/brainhack_jupyter_book#33]
I have added the bot and perhaps we can catch people's contributions related to this repo by replying to this issue for the time being.
Usage:
```
@all-contributors please add @<username> for <contributions>
```
[List of labels](https://allcontributors.org/docs/en/emoji-key)

**Comment by jsheunis** [comment]
@htwangtw just to check, adding contributions via all-contributor bot would be in parallel to allowing contributors to add themselves to the table in the Jupyter book itself? If that's the case and the latter is still required, I'll create a new issue to describe the proposed workflow and then we take it from there. Can you confirm if this sounds sensible?

**Comment by Remi-Gau** [comment]
> adding contributions via all-contributor bot would be in parallel to allowing contributors to add themselves to the table in the Jupyter book itself

that was my understanding at least. :wink:

**Comment by htwangtw** [comment]
@jsheunis Open your issue! If we can have all-encomposing solution I am happy to mark this as duplicate and kick the bot out :wink:

**Comment by htwangtw** [comment]
Move the discussion to #38
